src/storage/src/table/batch_table/storage_table.rs

```diff
@@ -846,6 +863,9 @@ struct StorageTableInnerIterInner<S: StateStore, SD: ValueRowSerde> {
 
     mapping: Arc<ColumnMapping>,
 
+    /// The index of system column `_rw_timestamp` in the output columns.
+    epoch_idx: Option<usize>,
```

Comment:
Move the special handling logic to `row_serde` means we need to change the deserializer interface from
```
pub trait ValueRowDeserializer: Clone {
    fn deserialize(&self, encoded_bytes: &[u8]) -> Result<Vec<Datum>>;
}
```
to 
```
pub trait ValueRowDeserializer: Clone {
    fn deserialize(&self, key_encoded_bytes: &[u8], value_encoded_bytes: &[u8],) -> Result<Vec<Datum>>;
}
```
It will also bring us to move `key_output_indices` and `value_output_indices` handling logic into `row_serde` (why not?).